client/src/test/scala/dwolla/cloudflare/FakeCloudflareService.scala

```diff
@@ -363,6 +365,21 @@ class FakeCloudflareService(authorization: CloudflareAuthorization) extends Http
       } yield res
   }
 
+  def listLogpushJobs(zoneId: String, responseBody: String) = HttpService[IO] {
+    case GET -> Root / "client" / "v4" / "zones" / id / "logpush" / "jobs" if id == zoneId ⇒
+      okWithJson(responseBody)
```

Comment:
All of these new fake services respond exclusively with JSON output. I would think about changing `responseBody` to be an `io.circe.Json` object, and then change `okWithJson(responseBody)` to `Ok(responseBody)`.

The source of the `responseBody` strings is JSON literals already, so right now we're going `String => Json => String => Json => String`. (The first `String => Json` occurs at compile-time as part of the JSON literal.) This would be simplified to `String => Json => String` by making this type replacement.